sbt/src/sbt-test/tests/fork-parallel/test

```diff
@@ -1,6 +1,5 @@
-# https://github.com/sbt/sbt/issues/3545
-# > test
-# -> check
+> test
+-> check
```

Comment:
fork-parallel failed 3 times in a row on Travis:

```
[error] x tests / fork-parallel 
[error]  Cause of test exception: {line 2}  Command succeeded but failure was expected
```

Reply:
Let's drop the relevant restore commit, and drop "Fixes #3545" from the PR desc, and merge.

Reply:
Yea. That what I was thinking too.